[browser][coreCLR] Wasm.Console.Node.Sample - undefined symbol: SystemInteropJS_GetManagedStackTrace

Seeing this on multiple PRs:
```
2026-06-09T19:58:42.0544708Z   Compiling generated managed-to-native sources with emcc -O2 ...
2026-06-09T19:58:42.5837769Z   [1/3] callhelpers-pinvoke.cpp -> callhelpers-pinvoke.o [took 0.52s]
2026-06-09T19:58:42.6555515Z   [2/3] callhelpers-interp-to-managed.cpp -> callhelpers-interp-to-managed.o [took 0.60s]
2026-06-09T19:58:42.7308959Z   [3/3] callhelpers-reverse.cpp -> callhelpers-reverse.o [took 0.67s]
2026-06-09T19:58:42.7432963Z   Linking CoreCLR WASM with initial memory 33554432 bytes.
2026-06-09T19:58:42.7435788Z   Linking with emcc -O2. This may take a while ...
2026-06-09T19:58:44.1657941Z wasm-ld : error : C:\Users\CLOUDT~1\AppData\Local\Temp\tmpqm0a9ffblibemscripten_js_symbols.so: undefined symbol: SystemInteropJS_GetManagedStackTrace. Required by BrowserHost_ExternalAssemblyProbe [D:\a\_work\1\s\src\mono\sample\wasm\console-node\Wasm.Console.Node.Sample.csproj]
2026-06-09T19:58:44.1659111Z wasm-ld : error : C:\Users\CLOUDT~1\AppData\Local\Temp\tmpqm0a9ffblibemscripten_js_symbols.so: undefined symbol: SystemInteropJS_CallDelegate. Required by BrowserHost_ExternalAssemblyProbe [D:\a\_work\1\s\src\mono\sample\wasm\console-node\Wasm.Console.Node.Sample.csproj]
2026-06-09T19:58:44.1660028Z wasm-ld : error : C:\Users\CLOUDT~1\AppData\Local\Temp\tmpqm0a9ffblibemscripten_js_symbols.so: undefined symbol: SystemInteropJS_CompleteTask. Required by BrowserHost_ExternalAssemblyProbe [D:\a\_work\1\s\src\mono\sample\wasm\console-node\Wasm.Console.Node.Sample.csproj]
2026-06-09T19:58:44.1660942Z wasm-ld : error : C:\Users\CLOUDT~1\AppData\Local\Temp\tmpqm0a9ffblibemscripten_js_symbols.so: undefined symbol: SystemInteropJS_ReleaseJSOwnedObjectByGCHandle. Required by BrowserHost_ExternalAssemblyProbe [D:\a\_work\1\s\src\mono\sample\wasm\console-node\Wasm.Console.Node.Sample.csproj]
2026-06-09T19:58:44.1661809Z wasm-ld : error : C:\Users\CLOUDT~1\AppData\Local\Temp\tmpqm0a9ffblibemscripten_js_symbols.so: undefined symbol: SystemInteropJS_BindAssemblyExports. Required by BrowserHost_ExternalAssemblyProbe [D:\a\_work\1\s\src\mono\sample\wasm\console-node\Wasm.Console.Node.Sample.csproj]
2026-06-09T19:58:44.1662677Z wasm-ld : error : C:\Users\CLOUDT~1\AppData\Local\Temp\tmpqm0a9ffblibemscripten_js_symbols.so: undefined symbol: SystemInteropJS_CallJSExport. Required by BrowserHost_ExternalAssemblyProbe [D:\a\_work\1\s\src\mono\sample\wasm\console-node\Wasm.Console.Node.Sample.csproj]
2026-06-09T19:58:44.2031904Z emcc : error : 'D:/a/_work/1/s/src/mono/browser/emsdk/bin\wasm-ld.exe -o D:\a\_work\1\s\artifacts\obj\mono\Wasm.Console.Node.Sample\browser.wasm.Release\wasm\for-publish\dotnet.native.wasm D:\a\_work\1\s\artifacts\obj\mono\Wasm.Console.Node.Sample\browser.wasm.Release\wasm\for-publish\callhelpers-pinvoke.o D:\a\_work\1\s\artifacts\obj\mono\Wasm.Console.Node.Sample\browser.wasm.Release\wasm\for-publish\callhelpers-reverse.o D:\a\_work\1\s\artifacts\obj\mono\Wasm.Console.Node.Sample\browser.wasm.Release\wasm\for-publish\callhelpers-interp-to-managed.o D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libBrowserHost.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libcoreclr_static.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libnativeresourcestring.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libgcinfo_unix_wasm.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libcoreclrminipal.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libcoreclrpal.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libminipal.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libSystem.Native.Browser.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libSystem.Runtime.InteropServices.JavaScript.Native.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libSystem.Native.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libSystem.Globalization.Native.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libicuuc.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libicui18n.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libicudata.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libSystem.IO.Compression.Native.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libz.a D:\a\_work\1\s\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\libSystem.Native.TimeZoneData.a --export=__stack_pointer --error-limit=0 -LD:\a\_work\1\s\src\mono\browser\emsdk\emscripten\cache\sysroot\lib\wasm32-emscripten -lGL-getprocaddr -lal -lhtml5 -lbulkmemory -lstubs -lc -ldlmalloc -lcompiler_rt-wasm-sjlj -lc++-except -lc++abi-except -lunwind-except -lsockets -mllvm -combiner-global-alias-analysis=false -mllvm -wasm-enable-sjlj -mllvm -disable-lsr -mllvm -wasm-enable-eh -mllvm -exception-model=wasm C:\Users\CLOUDT~1\AppData\Local\Temp\tmpqm0a9ffblibemscripten_js_symbols.so --export=free --export=htons --export=malloc --export=sbrk --export=memalign --export=posix_memalign --export=memset --export=ntohs --export=stackAlloc --export=stackRestore --export=stackSave --export=__stack_pointer --export=stackAlloc --export=__funcs_on_exit --export=__trap --export=__wasm_call_ctors --export=BrowserHost_CreateHostContract --export=BrowserHost_InitializeDotnet --export=BrowserHost_ExecuteAssembly --export=BrowserHost_ShutdownDotnet --export=SystemInteropJS_GetManagedStackTrace --export=SystemInteropJS_CallDelegate --export=SystemInteropJS_CompleteTask --export=SystemInteropJS_ReleaseJSOwnedObjectByGCHandle --export=SystemInteropJS_BindAssemblyExports --export=SystemInteropJS_CallJSExport --export=SystemJS_ExecuteTimerCallback --export=SystemJS_ExecuteBackgroundJobCallback --export=SystemJS_ExecuteFinalizationCallback --export=SystemJS_ExecuteDiagnosticServerCallback --export=GetDotNetRuntimeContractDescriptor --export=_exit --export=stackSave --export=stackRestore --export-if-defined=__start_em_asm --export-if-defined=__stop_em_asm --export-if-defined=__start_em_lib_deps --export-if-defined=__stop_em_lib_deps --export-if-defined=__start_em_js --export-if-defined=__stop_em_js --export-if-defined=fflush --export-table --growable-table -z stack-size=2097152 --max-memory=2147483648 --initial-memory=33554432 --no-entry --table-base=1 --global-base=1024' failed (returned 1) [D:\a\_work\1\s\src\mono\sample\wasm\console-node\Wasm.Console.Node.Sample.csproj]
2026-06-09T19:58:44.2280521Z D:\a\_work\1\s\src\mono\browser\build\BrowserWasmApp.CoreCLR.targets(643,5): error MSB3073: The command ""D:\a\_work\1\s\src\mono\browser\emsdk\emscripten\emcc" -fwasm-exceptions -msimd128 "@D:\a\_work\1\s\artifacts\obj\mono\Wasm.Console.Node.Sample\browser.wasm.Release\wasm\for-publish\emcc-link.rsp"" exited with code 1. [D:\a\_work\1\s\src\mono\sample\wasm\console-node\Wasm.Console.Node.Sample.csproj]
2026-06-09T19:58:44.3561395Z 
2026-06-09T19:58:44.3629303Z Build FAILED.
```

## Build Information
Build: https://dev.azure.com/dnceng-public/public/_build/results?buildId=1456473&view=results
Build error leg or test failing: browser-wasm windows checked CoreCLR_ReleaseLibraries
<!-- Error message template  -->
## Error Message

Fill the error message using [step by step known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Build%20Analysis/KnownIssueJsonStepByStep.md).

<!-- Use ErrorMessage for String.Contains matches. Use ErrorPattern for regex matches (single line/no backtracking). Set BuildRetry to `true` to retry builds with this error. Set ExcludeConsoleLog to `true` to skip helix logs analysis. -->

```json
{
  "ErrorMessage": "undefined symbol: SystemInteropJS_GetManagedStackTrace",
  "BuildRetry": false,
  "ExcludeConsoleLog": false
}
```

<!-- Known issue validation start -->
 ### Known issue validation
**Build: :mag_right:** https://dev.azure.com/dnceng-public/public/_build/results?buildId=1456473
**Error message validated:** `[undefined symbol: SystemInteropJS_GetManagedStackTrace`]
**Result validation:** :white_check_mark: Known issue matched with the provided build.
**Validation performed at:** 6/10/2026 11:30:22 AM UTC
<!-- Known issue validation end -->
<!--Known issue error report start -->

### Report
|Build|Repository|Step Name|Console log|Pull Request|
|---|---|---|---|---|
|[1461360](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1461360)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1461360/logs/419)|dotnet/runtime#129315|
|[1461350](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1461350)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1461350/logs/1073)|dotnet/runtime#129299|
|[1461046](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1461046)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1461046/logs/1730)|dotnet/runtime#129305|
|[1461072](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1461072)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1461072/logs/918)|dotnet/runtime#129043|
|[1460856](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460856)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460856/logs/1411)|dotnet/runtime#129316|
|[1460901](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460901)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460901/logs/1482)||
|[1459317](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459317)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459317/logs/1422)|dotnet/runtime#127760|
|[1460819](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460819)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460819/logs/602)|dotnet/runtime#78746|
|[1460768](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460768)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460768/logs/1851)|dotnet/runtime#129315|
|[1460790](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460790)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460790/logs/618)|dotnet/runtime#129304|
|[1460701](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460701)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460701/logs/1582)|dotnet/runtime#129034|
|[1460733](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460733)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460733/logs/571)|dotnet/runtime#129201|
|[1460706](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460706)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460706/logs/814)|dotnet/runtime#129278|
|[1460667](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460667)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460667/logs/498)|dotnet/runtime#129314|
|[1460644](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460644)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460644/logs/534)|dotnet/runtime#129312|
|[1460629](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460629)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460629/logs/574)|dotnet/runtime#129309|
|[1460625](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460625)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460625/logs/535)|dotnet/runtime#129140|
|[1460598](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460598)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460598/logs/610)|dotnet/runtime#129091|
|[1460589](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460589)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460589/logs/623)|dotnet/runtime#128658|
|[1460411](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460411)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460411/logs/2090)|dotnet/runtime#129083|
|[1460523](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460523)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460523/logs/1249)|dotnet/runtime#129299|
|[1460485](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460485)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460485/logs/573)|dotnet/runtime#128912|
|[1460462](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460462)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460462/logs/550)|dotnet/runtime#129019|
|[1460458](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460458)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460458/logs/593)|dotnet/runtime#129304|
|[1459990](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459990)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459990/logs/1411)|dotnet/runtime#129253|
|[1460431](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460431)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460431/logs/527)|dotnet/runtime#128795|
|[1460426](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460426)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460426/logs/1136)|dotnet/runtime#129197|
|[1460416](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460416)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460416/logs/589)|dotnet/runtime#128927|
|[1460390](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460390)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460390/logs/1280)|dotnet/runtime#128420|
|[1460384](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460384)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460384/logs/619)|dotnet/runtime#129309|
|[1459877](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459877)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459877/logs/1389)|dotnet/runtime#129255|
|[1459525](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459525)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459525/logs/1371)|dotnet/runtime#129285|
|[1460277](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460277)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460277/logs/470)|dotnet/runtime#129271|
|[1460273](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460273)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460273/logs/1099)||
|[1460236](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460236)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460236/logs/474)|dotnet/runtime#129200|
|[1460188](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460188)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460188/logs/992)|dotnet/runtime#129299|
|[1460180](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460180)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460180/logs/437)|dotnet/runtime#129300|
|[1460155](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460155)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460155/logs/410)|dotnet/runtime#129248|
|[1458627](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458627)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458627/logs/1608)|dotnet/runtime#126720|
|[1460148](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460148)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460148/logs/393)|dotnet/runtime#129082|
|[1460142](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460142)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460142/logs/461)|dotnet/runtime#129297|
|[1460096](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460096)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460096/logs/438)|dotnet/runtime#129306|
|[1460078](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460078)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460078/logs/428)|dotnet/runtime#129206|
|[1460058](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460058)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460058/logs/384)|dotnet/runtime#129141|
|[1460068](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460068)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460068/logs/359)|dotnet/runtime#129305|
|[1460022](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1460022)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1460022/logs/471)|dotnet/runtime#129303|
|[1459999](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459999)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459999/logs/432)|dotnet/runtime#129304|
|[1458751](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458751)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458751/logs/3010)|dotnet/runtime#129197|
|[1459493](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459493)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459493/logs/1502)|dotnet/runtime#128384|
|[1459800](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459800)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459800/logs/473)|dotnet/runtime#129145|
|[1459790](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459790)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459790/logs/564)|dotnet/runtime#129272|
|[1459744](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459744)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459744/logs/957)|dotnet/runtime#129293|
|[1459736](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459736)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459736/logs/384)|dotnet/runtime#129292|
|[1458800](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458800)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458800/logs/1779)|dotnet/runtime#129268|
|[1459711](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459711)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459711/logs/472)|dotnet/runtime#129269|
|[1459377](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459377)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459377/logs/1417)|dotnet/runtime#129249|
|[1459280](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459280)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459280/logs/1556)|dotnet/runtime#123138|
|[1459652](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459652)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459652/logs/489)|dotnet/runtime#129261|
|[1459554](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459554)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459554/logs/506)|dotnet/runtime#127520|
|[1459013](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459013)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459013/logs/1461)|dotnet/runtime#128906|
|[1456366](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1456366)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1456366/logs/2456)|dotnet/runtime#129183|
|[1459033](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459033)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459033/logs/1903)|dotnet/runtime#129278|
|[1458883](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458883)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458883/logs/1512)|dotnet/runtime#129141|
|[1459205](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1459205)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1459205/logs/1464)||
|[1458757](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458757)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458757/logs/1462)|dotnet/runtime#129267|
|[1458391](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458391)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458391/logs/1475)|dotnet/runtime#129248|
|[1458966](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458966)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458966/logs/604)|dotnet/runtime#128384|
|[1458413](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458413)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458413/logs/2643)|dotnet/runtime#129080|
|[1458932](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458932)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458932/logs/558)|dotnet/runtime#129203|
|[1458912](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458912)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458912/logs/591)|dotnet/runtime#129272|
|[1458859](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458859)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458859/logs/595)|dotnet/runtime#129271|
|[1458355](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458355)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458355/logs/1481)|dotnet/runtime#129189|
|[1458809](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458809)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458809/logs/574)|dotnet/runtime#129269|
|[1458690](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458690)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458690/logs/515)|dotnet/runtime#129201|
|[1458667](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458667)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458667/logs/667)|dotnet/runtime#129034|
|[1458649](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458649)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458649/logs/661)|dotnet/runtime#129206|
|[1458073](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458073)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458073/logs/1613)|dotnet/runtime#129058|
|[1458593](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458593)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458593/logs/493)|dotnet/runtime#125436|
|[1458561](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458561)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458561/logs/473)|dotnet/runtime#129059|
|[1458536](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458536)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458536/logs/537)|dotnet/runtime#129261|
|[1458456](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458456)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458456/logs/468)|dotnet/runtime#129029|
|[1458431](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458431)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458431/logs/1171)||
|[1458378](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458378)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458378/logs/450)|dotnet/runtime#129256|
|[1458364](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458364)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458364/logs/480)|dotnet/runtime#129255|
|[1458326](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458326)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458326/logs/471)|dotnet/runtime#129202|
|[1458308](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458308)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458308/logs/452)|dotnet/runtime#129253|
|[1458240](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458240)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458240/logs/464)|dotnet/runtime#129249|
|[1458217](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458217)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458217/logs/877)|dotnet/runtime#129043|
|[1458179](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458179)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458179/logs/372)|dotnet/runtime#129205|
|[1457163](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1457163)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1457163/logs/2074)|dotnet/runtime#129083|
|[1458015](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1458015)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1458015/logs/520)|dotnet/runtime#129240|
|[1457995](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1457995)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1457995/logs/641)|dotnet/runtime#128384|
|[1457861](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1457861)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1457861/logs/1045)|dotnet/runtime#128365|
|[1457851](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1457851)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1457851/logs/1023)|dotnet/runtime#128604|
|[1457573](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1457573)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1457573/logs/1407)|dotnet/runtime#129157|
|[1457823](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1457823)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1457823/logs/528)|dotnet/runtime#129199|
|[1457799](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1457799)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1457799/logs/488)|dotnet/runtime#127520|
|[1457784](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1457784)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1457784/logs/1022)|dotnet/runtime#129221|
|[1457474](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1457474)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1457474/logs/1407)|dotnet/runtime#128950|
|[1457299](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1457299)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1457299/logs/2797)||
Displaying 100 of 108 results
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|54|108|108|
<!--Known issue error report end -->

--- a/src/native/corehost/browserhost/libBrowserHost.footer.js
+++ b/src/native/corehost/browserhost/libBrowserHost.footer.js
@@ -29,7 +29,6 @@ function libBrowserHostFactory() {
     ];
     let commonDeps = [
         "$DOTNET",
-        "$DOTNET_INTEROP",
         "$ENV",
         "$FS",
         "$libBrowserHostFn",

